package.json

```diff
@@ -23,10 +23,9 @@
   },
   "homepage": "https://github.com/AnIdiotsGuide/guidebot#readme",
   "dependencies": {
-    "@discordjs/builders": "^0.2.0",
     "@sapphire/time-utilities": "^1.3.8",
```

Comment:
`@sapphire/time-utilities` can be updated to latest (`1.7.5`) along with other deps